docs/zetaclient/hotkey_tss_passwords.md

```diff
@@ -0,0 +1,34 @@
+## Hot Key and TSS key-share Passwords
+
+### Zetaclient
+Previously there were two environment variables being used to store passwords encrypting the tss key file and local operator keyring file.
+
+* HOTKEY_PASSWORD
+* TSS_FRAGMENT_SEED
+
+With this new change, these variables will no longer be valid.
+Instead, a series of prompts will appear asking for passwords using STDIN during the startup process.
+
+* Hot Key password
+* TSS Key share password
+
+If your key files are already encrypted, you can use the same passwords you provided in the environment variables.
+
+*****It's extremely important to take note of these passwords or commit them to memory.**
```

Comment:
If they forget about the passwords, they can still generate a encrypted version of the files based on copies of the original plaintext files using some tool?

Reply:
Yes, if they have the plaintext. In most cases they won't have this because the file gets encrypted during keygen if they had their password or environment variable set up.

There is another PR for a command to encrypt the plaintext json.